robobase/envs/wrappers/rescale_from_tanh.py

```diff
@@ -218,6 +218,7 @@ def __init__(
 
     @staticmethod
     def transform_from_tanh(action, action_stats, min_max_margin):
+        action = action.clip(-1.0, 1.0)
```

Comment:
Actually this is because the action head of ACT is just a simple linear layer, which doesn't transform policy output to [-1, 1], and it may go out of bounds during execution so we have to clip it.

Reply:
Wouldn't it be better to clip the action in `def act`?

Reply:
@richielo  Would this break Genima?

Reply:
Maybe it could be better to keep this line, and don't modify ACT as we're not sure if it will break genima or not

Reply:
I actually think it would be better to leave this line here. As we always assume actions to be bounded by [-1, 1] given transformations in robobase.